lib/maruku/input/mdline.rb

```diff
@@ -51,8 +51,8 @@ def line_md_type
       return :header3        if self =~ /\A(#)+\s*\S+/
       # at least three asterisks/hyphens/underscores on a line, and only whitespace
       return :hrule          if self =~ /\A(\s*[\*\-_]\s*){3,}\z/
-      return :ulist          if self =~ /\A[ ]{0,3}([\*\-\+])\s+.*/
-      return :olist          if self =~ /\A[ ]{0,3}\d+\.\s+.*/
+      return :ulist          if self =~ /^[ ]{0,1}([\*\-\+])\s+.*\w+/
+      return :olist          if self =~ /^[ ]{0,1}\d+\.\s.*\w+/
```

Comment:
Why would you want to be the only Markdown implementation that [does this wrong](http://johnmacfarlane.net/babelmark2/?normalize=1&text=No+indent%0A%0A*+One%0A*+two%0A%0AIndent+1+space%0A%0A+*+One%0A+*+Two%0A%0AIndent+2+spaces%0A%0A++*+One%0A++*+Two%0A%0AIndent+3+spaces%0A%0A+++*+One%0A+++*+Two)?

Reply:
oops!  Didn't mean to open this PR here :)

If you're curious - we have 11 years worth of data and millions of users that expect this behavior. I'm replacing an ancient version of Maruku and adjusting bhollis' project (it's great that someone took over this project) to match.

Reply:
(...but now that you point this out, I can probably get away with doing this right, there aren't many documents in my corpus that are affected)